governance/pyth_staking_sdk/src/pdas.ts

```diff
@@ -0,0 +1,37 @@
+import { PublicKey } from "@solana/web3.js";
+import {
+  INTEGRITY_POOL_PROGRAM_ADDRESS,
+  STAKING_PROGRAM_ADDRESS,
+} from "./constants";
+
+export const getConfigAddress = () => {
```

Comment:
seems inconsistent that this one doesn't return just the Pubkey but also the bump

Reply:
will deal with this later